script/index.coffee

```diff
@@ -774,6 +776,10 @@ api.wrap = (user, main=true) ->
           if not user.achievements.mountMasterCount
             user.achievements.mountMasterCount = 0
           user.achievements.mountMasterCount++
+          if giveTriadBingo
```

Comment:
@deilann is this acceptable to you? 
+ Set `giveTriadBingo = true`
+ for every pet, check if `pet == -1`, and if so set `giveTriadBingo = false`. 
+ if `giveTriadBingo == true` at the end, then increase count of triadBingoCount

You can't use `releaseBoth()` without having all 90 mounts, so there's no need to check for that.